### DISCUSSION TITLE
the parallel computing function of Aeolis

### DISCUSSION THREAD

**Discussion by FDDZG** [openearth/aeolis-python#231] [Q&A]
Hello Professor sired, I have a question about Aeolis program to consult you, that is, how to enable the parallel computing function of Aeolis program. Sorry for your trouble

**Comment by Sierd** [comment]
Hi @FDDZG , sorry for replying a bit late. I have been busy in the field a bit. 

The parallel computing function of AeoLiS became obsolete with the introduction of the steady state solver. 
This steady state solver allows us to avoid an expensive matrix inversion which resulted in a very efficient calculation (especially for larger domains) reducing the calculation time to the point we did not need a parallel solver anymore. 

If you wish to speed up your model I would be happy to help.
